lib/ansible/config/base.yml

```
@@ -234,8 +234,10 @@ COLLECTIONS_PATHS:
   type: pathspec
   env:
   - {name: ANSIBLE_COLLECTIONS_PATHS}
+  - {name: ANSIBLE_COLLECTIONS_PATH}
```

I'm not sure whether this works:
```suggestion
  - {name: ANSIBLE_COLLECTIONS_PATH, version_added: '2.10'}
```

---

also, deprecate the old entry

---

In that case, the docs that reference the old one should be updated too. (I know of at least one mention in the dev guide. And maybe ansible-test is also using it somewhere.)

---

I've added commits to change all existing references to the singular version.  I've also deprecated the plural version.